tests/e2e/rudder-npm-js/src/analytics/rudder.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * This client was automatically generated by RudderTyper. ** Do Not Edit **
+ */
+
+import type { RudderAnalytics, RudderAnalyticsPreloader } from '@rudderstack/analytics-js/*';
+
+/**
+ * The analytics.js snippet should be available via window.analytics.
+ * You can install it by following instructions at: https://docs.rudderstack.com/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk#2-installing-the-rudderstack-javascript-sdk
+ */
+declare global {
+  interface Window {
+    rudderanalytics: RudderAnalytics | RudderAnalyticsPreloader | undefined;
+  }
+}
```

Comment:
We should update the template to the latest.
```
declare global {
  interface Window {
    rudderanalytics: RudderAnalytics | RudderAnalyticsPreloader | undefined;
    RudderStackGlobals: IRudderStackGlobals;
    RudderSnippetVersion?: string;
  }
}
```

Reply:
Not addressing it right now. When we will fix the type-related issues we can address this.